### ISSUE TITLE
Custom Keyboard control on top of WebView2

### ISSUE THREAD

**Issue by tobias-tengler** [MicrosoftEdge/WebView2Feedback#600]
Our WPF App runs on a Touch Display without a Virtual Keyboard. 
We've built our own Keyboard Control we've displayed on top of the CefSharp browser using their `OnVirtualKeyboardRequested` event in the past.
At the moment we're trying out WebView2, but there does not appear to be a similar event, we could use for detecting, when a keyboard input is required.
How should I go about this with WebView2 or rather is there anything planned for a scenario like this?

P.S.: I've read that there are issues with displaying other controls on top of WebView2, due to rendering issues. Is our scenario doomed in that case?

**Comment by TheRisenPhoenix** [comment]
I'd be interested in a similar VirtualKeyboard-event as well

**Comment by pagoe-msft** [comment]
@tobias-tengler 

Yes, unfortunately laying a WPF virtual keyboard on top of a WPF WebView2 controls likely will run into some known airspace issues. However, we are investigating a fix for this.

I've added this as a feature request and placed it in our backlog.

**Comment by tobias-tengler** [comment]
We've come to the decision to not adapt WebView2, with this issue being the deciding factor. We'll be sticking to CefSharp.
I'm hoping this is addressed in the future and we can consider the switch again!

**Comment by champnic** [comment]
Thanks for the honesty @tobias-tengler. I'm sad you won't be using WebView2 for now, but it's really good to have this feedback and we hope we'll have support for the airspace issues soon.

**Comment by dcreurer** [comment]
We also have WPF application with a WebView2 control and a virtual keyboard. They are running in the same application, but in separate Windows, so maybe no issue with airspace. But we are looking for an approach to sending the keyboard input to the focused element in the WebView2 control.

**Comment by Eudritch** [comment]
Please let me know if anyone manages to send keyboard or mouse input into webview and how you did it.
